schema.graphql

```diff
@@ -117,4 +119,21 @@ type TraderDayData @entity {
   fee: BigInt! # trading fee aggregated per day for trader
   fundingPayment: BigInt! # funding payment aggregated per day for trader
   realizedPnl: BigInt! # total pnl aggregated per day for trader
+}
+
+type ReferralCode @entity {
+  id: ID!
+  referrer: Trader! # referrer address
+  referees: [Bytes!]! # referees
+  createdAt: BigInt! # created at
+}
+
+type ReferralCodeDayData @entity {
+  id: ID!
+  referralCode: ReferralCode!
+  date: BigInt!
+  tradingVolume: BigInt! # trading volume in notional value
+  fees: BigInt! # fees paid during day
+  newReferees: [Bytes!]! # a list of new referees added today
+  activeReferees: [Bytes!]! # a list of referees which have traded on the day
```

Comment:
same above comment for `Trader`?

Reply:
second that, both `newReferees` and `activeReferees`